pkg/cmdlets/commit/commit.go

```diff
@@ -73,7 +78,8 @@ func commitDataFromCommit(commit *mono_models.Commit, orgs []gmodel.Organization
 	if commit.Author != nil && orgs != nil {
 		username, err = usernameForID(*commit.Author, orgs)
 		if err != nil {
-			return commitData{}, locale.WrapError(err, "err_commit_print_username", "Could not determine username for commit author")
+			logging.Debug("Could not determine username for commit author '%s'. Using placeholder value '%s'.", *commit.Author, AuthorDeleted)
+			username = AuthorDeleted
```

Comment:
This logic should be contained within `userrnameForID`, or alternatively it should return an identifiable error so that we can fall back. `err != nil` could have any number of causes as far as this code is concerned.

Reply:
Fair enough.